master/pkg/schemas/expconf/schema_test.go

```diff
@@ -81,24 +84,41 @@ func (tc SchemaTestCase) CheckCompleteAs(t *testing.T) {
 	}
 }
 
-func (tc SchemaTestCase) CheckErrors(t *testing.T) {
-	if tc.Errors == nil {
+func (tc SchemaTestCase) CheckSanityErrors(t *testing.T) {
+	if tc.SanityErrors == nil {
 		return
 	}
+	checkErrors(tc, t, tc.SanityErrors, "sanity")
+}
+
+func (tc SchemaTestCase) CheckCompletenessErrors(t *testing.T) {
+	if tc.CompletenessErrors == nil {
+		return
+	}
+	checkErrors(tc, t, tc.CompletenessErrors, "completeness")
+}
+
+func checkErrors(tc SchemaTestCase, t *testing.T, errors *map[string][]string, testType string) {
```

Comment:
This seems like it should be a method on `tc SchemaTestCase` like the others.  (but lower-casing the `c` of `checkErrors` is good, because that distinguishes it from the other "higher-level" functions, the way that the lead `_` of `_run_errors` does in python)